prac_05/word.occurrences.py

```diff
@@ -0,0 +1,17 @@
+"""This program will count the occurrences of words in a string."""
+
+text = {}
```

Comment:
I suggest 99.9% of people seeing a variable called `text` would think it's a `string`.